### PR TITLE
feat(Stack): creating new Stack component

### PR DESCRIPTION
**Pull request by dominiquemb**

### CODE REVIEW

File: src/components/Stack/Stack.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+
+const Code = ({ children }: { children: ReactNode }): ReactElement => (
+    <code className="tw-bg-box-neutral tw-rounded tw-px-2 tw-text-box-neutral-inverse tw-text-s">{children}</code>
+);
+
+export const Stack: StoryFn<STACK_PROPS> = (args: STACK_PROPS) => {
+    const isVertical = args.direction === 'row' || args.direction === 'row-reverse';
+
+    return (
+        <StackComponent {...args} divider={<Divider vertical={isVertical} />}>
```
Comment (imoutaharik) [inline src/components/Stack/Stack.stories.tsx:88]: The Divider is enforced while it's an optional property. I would create different stories:
- Stack (No Divider)
- Stack with horizontal Divider
- Stack with vertical Divider

**Review by imoutaharik — CHANGES_REQUESTED** [review]

File: src/components/Stack/Stack.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+            This story shows only Box Alias Tokens in controls, but you can pass any color token class to define bg and
+            color properties.
+        </p>
+    </StackComponent>
+);
+StackWithBoxAliasToken.args = {
+    bg: 'tw-bg-box-neutral',
+};
+
+export const StackAsSpanElement: StoryFn<STACK_PROPS> = (args: STACK_PROPS) => (
```
Comment (imoutaharik) [inline src/components/Stack/Stack.stories.tsx:137]: That story doesn't seem to be very useful. As there is only one item within the Stack component, property like alignment or justify for example seems useless.

**Review by imoutaharik — CHANGES_REQUESTED** [review]

**Review by imoutaharik — CHANGES_REQUESTED** [review]

File: src/components/Stack/Stack.stories.tsx (hunk 2 above)
Comment (dominiquemb) [reply src/components/Stack/Stack.stories.tsx]: @imoutaharik I think I forgot to push the changes. The span story is gone now.

**Review by imoutaharik — COMMENTED** [review]
Thanks for making the changes! 🎉  This looks good to me.
But there is just one tiny detail before we can merge... don't forget to export the components in the `index.ts` file at the root of `components` folder. 😉

**Review by imoutaharik — CHANGES_REQUESTED** [review]
Last comment is still not addressed.

**Review by imoutaharik — APPROVED** [review]

**Pull request merged**